extensions/pkg/webhook/cmd/options.go

```diff
@@ -294,6 +303,45 @@ func (c *AddToManagerConfig) AddToManager(ctx context.Context, mgr manager.Manag
 		return nil, fmt.Errorf("could not create webhooks: %w", err)
 	}
 
+	if mergeShootWebhooksIntoSeedWebhooks {
+		clientConfig := func(in admissionregistrationv1.WebhookClientConfig) (admissionregistrationv1.WebhookClientConfig, error) {
+			var path string
+			if in.Service != nil {
+				path = ptr.Deref(in.Service.Path, "")
+			} else if u := in.URL; u != nil {
+				parsedURL, err := url.Parse(*u)
+				if err != nil {
+					return admissionregistrationv1.WebhookClientConfig{}, fmt.Errorf("failed to parse URL: %v", err)
```

Comment:
Not sure if the error will also include the URL or if it should be added here for context, e.g.

```suggestion
					return admissionregistrationv1.WebhookClientConfig{}, fmt.Errorf("failed to parse URL %q: %w", *u, err)
```

WDYT?